tests/test_edge_cases.py

```diff
@@ -1240,10 +1240,10 @@ class Model(BaseModel):
 
 def test_force_extra():
     class Model(BaseModel):
-        model_config = ConfigDict(extra='ignore')
+        model_config = ConfigDict(extra=Extra.ignore)
```

Comment:
Is there still a test that it coerces strings properly? I recognize other PRs are possibly changing this behavior, but I think that was the purpose of this test, and until we do change how `Extra` works, presumably we should still have a test that that works (for migration purposes or whatever; at least assuming it worked in v1, which I think it did). Even if it raises a deprecation warning.

Reply:
I’ll revert this change and rename the test